app/controllers/unaccompanied_controller.rb

```diff
@@ -374,7 +380,7 @@ def handle_step
 
       if !check_nationality_validity(params["unaccompanied_minor"]["adult_nationality"])
```

Comment:
Thinking out loud - I wonder if we should rename this method to `nationality_valid?`

Reply:
yeah I'll rename it to something even better, because all the method does is check whether the nationality exists in our list